frontend/src/generate-static-html.js

```diff
@@ -1,24 +1,43 @@
 
 import React from 'react';
 import ReactDOMServer from 'react-dom/server';
+import ReactMarkdown from 'react-markdown';
 
+import Loading from './app/components/Loading';
 
-function makeStaticString(headComponent, bodyComponent) {
+function makeStaticString(
```

Comment:
Maybe it is not too bad. It is big and unweildy, and could be refactored with better function names and arguments, but it works for now.